includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2443,6 +2443,58 @@ public function schedule_order_tracking( $order_id, $order = null ) {
 		}
 	}
 
+	/**
+	 * Create a payment intent without confirming the intent.
+	 *
+	 * @param WC_Order $order - Order based on which to create intent.
+	 * @param array    $payment_methods - A list of allowed payment methods. Eg. card, card_present.
+	 * @param ?string  $capture_method - Controls when the funds will be captured from the customer's account ("automatic" or "manual").
+	 * It must be "manual" for in-person (terminal) payments.
+	 *
+	 * @return array An array containing the status (succeeded/failed), id (intent ID), message (error message if any), and http code.
+	 *
+	 * @throws Exception - When an error occurs in intent creation.
+	 */
+	public function create_intent( WC_Order $order, array $payment_methods, string $capture_method ) {
```

Comment:
you could add the default value of `$capture_method` here and remove it from the method body.

Reply:
Nice. Done ✅